tools/cli/workflowCommands.go

```diff
@@ -384,7 +381,7 @@ func getPrintableMemo(memo *commonpb.Memo) string {
 	buf := new(bytes.Buffer)
 	for k, v := range memo.Fields {
 		var memo string
-		err := payload.Decode(v, &memo)
+		err := defaultDataConverter().FromPayload(v, &memo)
```

Comment:
Similar to my above comment, seems we can just call `dataconverter.GetCurrent()`. Or this has to be the default converter even if a user data converter has been provided?

Reply:
It should be the default converter. The SDK does not run memo through the user converter.